build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -22,6 +22,135 @@ data:
     & antrea-agent --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
   Run-AntreaOVS-Containerd.ps1: |
     $ErrorActionPreference = "Stop"
+    $AntreaVersion = "3.0.5-antrea.0"
+    $OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-$AntreaVersion-win64.zip"
+    $OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
+    $WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
```

Comment:
This line fetches the work directory based on script location, since the .ps1 script is part of yaml, we might not need this line as it would return void.

Reply:
All the scripts used in yaml file are present in /var/lib/antrea-windows , so this directory will be returned as output to $WorkDir